scripts/update-projects-version.py

```diff
@@ -0,0 +1,375 @@
+    else:
+        exit_error(f"Pattern {pattern} was not found in {file}")
+    # Verification if the changes were actually applied
+    if file.read_text().count(new) != number_of_changes:
+        exit_error(f"Unexpected changes in {file}")
+
+
+def update_charts_folder(next_chart_version: str) -> None:
+    for file, occurrences in CHARTS_FILES_TO_UPDATE.items():
+        replace_in_file(
+            file=file,
+            pattern=VERSION_PATTERN,
+            new=next_chart_version,
+            number_of_changes=occurrences,
+        )
+
+    run_command(f"helm dep update {PELORUS_CHARTS_FOLDER}")
+    tar_files = (PELORUS_CHARTS_FOLDER / "charts").glob("*.tgz")
+    for file in tar_files:
+        file.unlink()
+
+
+def add_replaces_to_csv(
+    new_version: str, cluster_service_version_file: Path, versions: List[str]
+) -> None:
+    if not cluster_service_version_file.exists():
+        exit_error(f"{cluster_service_version_file} does not exist.")
+    if new_version in versions:
+        # TODO move this to chart-test?
+        exit_error(f"version {new_version} already exists in Pelorus Operator tags.")
+
+    with open(cluster_service_version_file, encoding="utf-8") as file_content:
+        cluster_service_version_file_content = file_content.readlines()
+
+    cluster_service_version_file_content.append(
+        f"{YAML_TAB}replaces: pelorus-operator.v{versions[0]}\n"
+    )
+    cluster_service_version_file_content.append(f"{YAML_TAB}skips:\n")
+    for version in versions:
+        cluster_service_version_file_content.append(
+            f"{YAML_TAB*2}- pelorus-operator.v{version}\n"
+        )
+
+    with open(cluster_service_version_file, mode="w", encoding="utf-8") as file_content:
+        file_content.writelines(cluster_service_version_file_content)
+
+
+def update_operator_folder(next_operator_version: str, destination: Path) -> None:
+    logging.info("Creating operator init files")
+    # TODO run update command instead of init every time
+    run_command(
+        f"operator-sdk init --plugins=helm --domain={OPERATOR_DOMAIN} --project-name={OPERATOR_NAME}",
+        directory=destination,
+    )
+    logging.info("Creating operator api")
+    exporters_charts_in_operator = destination / "helm-charts/pelorus/charts/exporters"
+    exporters_charts_in_operator.mkdir(parents=True)
+    shutil.copytree(
+        PELORUS_EXPORTERS_FOLDER, exporters_charts_in_operator, dirs_exist_ok=True
+    )
+    run_command(
+        f"operator-sdk create api --helm-chart={PELORUS_CHARTS_FOLDER}",
+        directory=destination,
+    )
+    for file in exporters_charts_in_operator.parent.glob("*.tgz"):
+        file.unlink()
+
+    replace_in_file(
+        file=PELORUS_OPERATOR_FOLDER / "Makefile",
+        pattern=r"VERSION \?= 0\.0\.1",
+        new=f"VERSION ?= {next_operator_version}",
+        number_of_changes=1,
+    )
+    operator_image = f"quay.io/{OPERATOR_ORGANIZATION_NAME}/{OPERATOR_NAME}"
```

Comment:
move quay.io to something we can configure, I would like sometimes to push to local registry for example.